ev3dev/core.py

```diff
@@ -197,12 +197,12 @@ def _get_attribute(self, attribute, name):
     def _set_attribute(self, attribute, name, value):
         """Device attribute setter"""
         if self.connected:
-            if None == attribute:
-                attribute = self._attribute_file_open( name )
-            else:
-                attribute.seek(0)
-
             try:
+                if None == attribute:
```

Comment:
Do we follow PEP-8?  If so we should do `if attribute is None` here.

Reply:
Yeah, this is existing code. I'll update it.